### PR TITLE
Update information-technology-quiz.ts

### PR DESCRIPTION
**Pull request by astitva0011**
* About SPF 
* Updated: Information technology quiz

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/information-technology-quiz.ts [hunk 1]
```diff
@@ -459,6 +459,15 @@ const informationTechnologyQuiz = [
       "Layer 1 is the physical layer. There’s a lot of technology in Layer 1 - everything from physical network devices, cabling, to how the cables hook up to the devices.",
     Link: "https://www.freecodecamp.org/news/osi-model-networking-layers-explained-in-plain-english/",
   },
+  {
+    Question: "What is SPF? and how it works?",
```
Comment (jdwilkin4) [inline src/data/information-technology-quiz.ts:463]:
```suggestion
    Question: "What does SPF stand for?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by astitva0011** [comment]
> Thank you for your contribution!
> 
> Please review the requested changes. Once those changes have been implemented, then we can merge your PR into main. 👍

okay Thanks for the suggestions

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by astitva0011** [comment]
> Congrats on another contribution to our repository. 🎉 We look forward to future contributions. 👍

Thank you ma'am🥳

**Pull request merged**